app/javascript/components/base/base-table-component.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <table class="min-w-full divide-y divide-gray-200">
+    <thead class="justify-between">
+      <tr class="bg-yellow-500 border-4 border-yellow-500 py-1">
+        <th v-for="h in table.header">
+          <span class="text-white font-bold">{{ h }}</span>
+        </th>
+        <th v-if="dots" />
+      </tr>
+    </thead>
+    <tbody class="bg-gray-200">
+      <tr
+        v-for="element in table.body"
+        class="bg-white border-4 border-gray-200"
+      >
+        <td
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key